src/main/java/org/jenkinsci/test/acceptance/docker/fixtures/LdapContainer.java

```diff
@@ -38,7 +38,10 @@ public String getManagerPassword() {
      * @return default ldap connection details from current running docker LdapContainer.
      */
     public LdapDetails createDefault() {
-        return new LdapDetails(getHost(), getPort(), getManagerDn(), getManagerPassword(), getRootDn());
+        return new LdapDetails(addBracketsIfNeeded(getHost()), getPort(), getManagerDn(), getManagerPassword(), getRootDn());
```

Comment:
this is incorrect.  LDAPDetails is expecting a host `[22:33::12]` is not a host but an encapsulated ip6 host.  

if this is needed them something is using LDAPDetails incorrectly - possibly via `getHostWithPort` which is likely to be incorrect.

Reply:
I get your point, but AFAIK this `host` is to be used by docker fixtures to connect via `tcp` so finally it is an address and requires to be specified using brackets

Reply:
I do not follow your response,  IP uses bytes and it does not use `[]` in the address portion of the packets, so any `tcp` or `udp` connection is irrelevant.

Reply:
issue here is that LDAP plugin field is not wanting a hostname but a portion of hosename:port and optionally scheme (ldap://hostname)  as such the plugin itself is requireing this and the host is just missnamed.
thus not an issue in the test but production code.